src/Toml/BiMap.hs

```diff
@@ -198,16 +209,27 @@ _Natural :: BiMap Natural AnyValue
 _Natural = _NaturalInteger >>> _Integer
 
 _WordInteger :: BiMap Word Integer
-_WordInteger = BiMap (Just . toInteger) maybeInteger
-  where maybeInteger :: Integer -> Maybe Word
-        maybeInteger n
+_WordInteger = BiMap (Just . toInteger) maybeWord
+  where maybeWord :: Integer -> Maybe Word
+        maybeWord n
           | n < toInteger (minBound :: Word) = Nothing
           | n > toInteger (maxBound :: Word) = Nothing
           | otherwise                        = Just (fromIntegral n)
 
 _Word :: BiMap Word AnyValue
 _Word = _WordInteger >>> _Integer
 
+_IntInteger :: BiMap Int Integer
+_IntInteger = BiMap (Just . toInteger) maybeInt
+  where maybeInt :: Integer -> Maybe Int
+        maybeInt n
+          | n < toInteger (minBound :: Int) = Nothing
```

Comment:
With `TypeApplications` it looks neater: `minBound @Int` (but it's up to you 🙂)

Reply:
I didn't actually know that notation, thank you. I left it as is for the moment in case I manage to define the polymorphic function that alludes me at the moment.